DemoServer/Controllers/Demos/TextSearch/FTSQuerySearchBoosting/metadata.json

```diff
@@ -0,0 +1,123 @@
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/querying/boosting"
+        },
+        {
+            "Title": "Indexes boosting",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/boosting"
+        }
+        
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define a query",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* Define a query on the 'Employees' collection Using the Session `Query()` LINQ method syntax."
+            ],
+            "Assets": [
+                {
+                    "Title": "Query Overview Demo",
+                    "Type" : "demo",
+                    "Url"  : "/queries/query-overview"
+                },
+                {
+                    "Title": "LINQ Query & Method Syntax",
+                    "Type" : "link",
+                    "Url"  : "https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/query-syntax-and-method-syntax-in-linq"
+                }
+            ]
+        },
+        {
+            "Title": "Define search criteria",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* Use `Search()` to search for the specified terms on the specified document field.",
+                "",
+                "* Use `boost: boostValue` to set the boost value per term.<br>",
+                "  The higher the value, the higher the matching document will be listed in the results."
+            ],
+            "Assets": [
+                {
+                    "Title": "Querying boosting",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/querying/boosting"
+                },
+                {
+                    "Title": "Indexes boosting",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/boosting"
+                }
+            ]
+        },
+        {
+            "Title": "Execute the query",
+            "Slug": "step-3",
+            "DescriptionLines": [
+                "* Execute the query by calling `ToList()`.<br>",
+                "  An Auto-Index is created upon executing the query for the first time with 2 index fields:",
+                "    * _Notes_ - contains the original text from the indexed document field Notes.",
+                "",
+                "    * _Search(Notes)_ - contains terms from the Notes field that were tokenized by RavenDB's default analyzer (StandardAnalyzer).<br>",
+                "      If your text needs to be tokenized with a different analyzer, then you need to use an index instead of a dynamic query.",
+                "",
+                "* Variable 'employeesWithMatchingTerms' will hold the matching Employee document entities.<br>",
```

Comment:
Variable 'employeesWithMatchingTerms' -> 'employeesWithMatchingTerms' list?

Reply:
Not sure I follow
Did you mean you think the variable should be named: employeesWithMatchingTermsList ?
Can we leave it as is ?
The variable name starts with 'employees' meaning it is employees (plural) that have matching terms
(Usually when adding 'list' at the end it needs to turn into a singular noun, i.e. employeeList)

Also higher in the demo there is the List definition 
`List<Employee> employeesWithMatchingTerms;`

Reply:
Sorry for not being specific - proposed change was to use word "list" instead of "variable"